article/app/views/liveblog/dateBlock.scala.html

```diff
@@ -2,8 +2,8 @@
 
 @(date: Option[LiveBlogDate])
 
-@date.map { case LiveBlogDate(publishedDate, hhmm, ampm, gmt) =>
-    <time datetime="@publishedDate" data-relativeformat="med" class=" js-timestamp" itemprop="datePublished">@ampm <span class="timezone">@gmt</span></time>
+@date.map { case LiveBlogDate(date, hhmm, ampm, gmt) =>
+    <time datetime="@date" data-relativeformat="med" class=" js-timestamp" itemprop="datePublished">@ampm <span class="timezone">@gmt</span></time>
```

Comment:
it might be helpful to call this firstPublishedDate as well to help people find it

Reply:
Um.......... ok!

Reply:
Done